### PR TITLE
Update vocabulary to match AV1 1.0.0 specification

### PR DESCRIPTION
**Pull request by ocrete**
Some of the terms used are confusing because they don't match what the AV1 spec uses. Here are 2 patch that address obvious ones.

Addresses #49

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -270,9 +270,9 @@ NOTE: Other types of OBUs such as [=metadata OBUs=] could be present before the
 
 [=Delayed Random Access Points=] SHOULD be signaled using sample groups and the [=AV1ForwardKeyFrameSampleGroupEntry=].
 
-[=S Frames=] SHOULD be signaled using sample groups and the [=AV1SFrameSampleGroupEntry=].
+[=Switch Frames=] SHOULD be signaled using sample groups and the [=AV1SFrameSampleGroupEntry=].
```
Comment (tomfinegan) [inline index.bs]: s/AV1SFrameSampleGroupEntry/AV1SwitchFrameSampleGroupEntry/ please

**Comment by ocrete** [comment]
I updated my patch to also change that, I was afraid changing the "formal" name might break something.

**Comment by cconcolato** [comment]
LGTM except for the link names that miss the 's' in 'amplegroup' but the problem was there before...

**Comment by tomfinegan** [comment]
> LGTM except for the link names that miss the 's' in 'amplegroup' but the problem was there before...

I'll handle that in a follow up.

**Pull request merged**